tools/cldr-apps/js/src/views/VettingSummary.vue

```diff
@@ -13,68 +13,76 @@
 </template>
 
 <script>
+import * as cldrAjax from "../esm/cldrAjax.js";
+
+const SECONDS_IN_MS = 1000;
+
+// timeouts
+const RETRY_ON_FETCH_ERR = 15 * SECONDS_IN_MS; // When we couldn't talk to server
+const NORMAL_RETRY = 5 * SECONDS_IN_MS; // "Normal" retry: starting or about to start
+const BUSTED_RETRY = 60 * SECONDS_IN_MS; // ST is down
```

Comment:
maybe these should go into `cldrAjax.js`?

Reply:
think these constants are in a couple of files now

Reply:
that might be good -- note btw that 5 seconds is pretty frequent; compare the all-row get row which is only every 15 seconds; also i made a note in today's agenda about a bug where the complete data is getting re-downloaded every 5 seconds, redundantly